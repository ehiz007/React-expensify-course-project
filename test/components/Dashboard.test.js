import React from 'react'
import {shallow} from 'enzyme'
import Dashboard from '../../src//components/Dashboard'
import toJSON from 'enzyme-to-json'

test('should display the dashboard component', () => {
    const wrapper = shallow(<Dashboard />)
    expect(toJSON(wrapper)).toMatchSnapshot()
})