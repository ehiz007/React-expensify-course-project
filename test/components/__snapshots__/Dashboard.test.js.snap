// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should display the dashboard component 1`] = `
<div>
  <Connect(ExpenseSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
